src/main/java/org/fundacionjala/core/Environment.java

```diff
@@ -18,23 +20,26 @@
 
     private String user;
     private String pass;
+    private String credUser = "$['";
+    private String credPass = "$['";
```

Comment:
this variable should be have better descriptive name like passwordCredentials or password